unconvert.go

```diff
@@ -99,53 +101,44 @@ func (e *editor) rewrite(f *ast.Expr) {
 	delete(e.edits, pos)
 }
 
-func print(name string, edits editSet) {
-	if len(edits) == 0 {
-		return
-	}
-
-	buf, err := ioutil.ReadFile(name)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	for pos := range edits {
+func print(edits []token.Position) {
+	for _, pos := range edits {
 		fmt.Printf("%s:%d:%d: unnecessary conversion\n", pos.Filename, pos.Line, pos.Column)
 		if *flagV {
-			line := lineForOffset(buf, pos.Offset)
-			fmt.Printf("%s\n", line)
-			fmt.Printf("%s^\n", rub(line[:pos.Column-1]))
+			line := lineForPos(pos)
```

Comment:
Seems unfortunate to need to re-read the source file for each warning message.  What was wrong with the previous method of slurping it into memory?